src/react-app-env.d.ts

```diff
@@ -8,8 +8,11 @@ interface Window {
   // walletLinkExtension is injected by the Coinbase Wallet extension
   walletLinkExtension?: any
   ethereum?: {
-    // value that is populated and returns true by the Coinbase Wallet mobile dapp browser
+    // set by the Coinbase Wallet mobile dapp browser
     isCoinbaseWallet?: true
+    // set by the Brave browser when using built-in wallet
+    isBraveWallet?: true
+    // set by the MetaMask browser extension (in Brave browser when using built-in wallet too)
```

Comment:
Wondering if we should change the comment to:
```
// set by the MetaMask browser extension (also set by Brave browser when using built-in wallet)
```
The comment in its current form seems somewhat confusing